benchmarks/nel/configs/nel.cfg

```diff
@@ -41,8 +41,10 @@ incl_prior = true
 labels_discard = []
 
 [components.entity_linker.get_candidates]
-@misc = "EmbeddingGetCandidates.v1"
+@misc = "FuzzyStringGetCandidates.v1"
```

Comment:
It's kind of unfortunate that the project doesn't show the difference between these two candidate generators in the `evaluate` step. I wonder whether that could be done by dynamically looping through the candidate generator options/configs, replacing them in the config at inference time and then show more detailed results instead of just "Candidate Gen." in the final tables?

That's assuming that we're training on gold entities, because otherwise the training is also affected by this choice.

Reply:
I was thinking about that too, but discarded the idea because I thought it would make things messier than necessary by moving away from the one config -> one run idea. Not ruling it out though. 

Alternative idea: store evaluation data in a machine-readable format and add a step comparing the performance from multiple collected runs somehow? 
So we'd have to run the whole workflow multiple times and then could do `spacy project run compare_evaluations` - or something similar - and get a table comparing the results from different runs (e.g. different candidate generators).

Reply:
Yea, that'd be nice. And we could just store the different configs rather than automatically generating them, maybe, because then the configs can be more easily copy/pasted by users.

Reply:
Makes sense for a benchmark, I think. I'd suggest doing that in a different PR though, if that's ok with you?

Reply:
Sure!